credential-manager/test/CredentialManager/Scripts/HotNFT/VoteSpec.hs

```diff
@@ -37,38 +37,32 @@ spec = do
     "Invariant V2: Vote ignores duplicate signers in voting group"
     invariantV2DuplicateVoting
   prop
-    "Invariant V3: Vote fails if gov action ID match"
-    invariantV3GovActionIdMismatch
+    "Invariant V3: Vote fails if the voter doesn't match"
+    invariantV3VoterMismatch
   prop
-    "Invariant V4: Vote fails if the voter doesn't match"
-    invariantV4VoterMismatch
+    "Invariant V4: Vote fails if voting list is empty"
+    invariantV4EmptyVoting
   prop
-    "Invariant V5: Vote fails if the vote doesn't match"
-    invariantV5VoteMismatch
+    "Invariant V5: Vote fails if extra votes are present"
+    invariantV5ExtraVotes
```

Comment:
This is no longer a valid invariant - the point of the change is to allow multiple votes per transaction.